.github/workflows/ci.yaml

```diff
@@ -71,4 +71,4 @@ jobs:
         run: poetry install --sync
 
       - name: Run tests
-        run: poetry run pytest
+        run: poetry run pytest -m "not jobs"
```

Comment:
You need to register this marker in the `pytest` pyproject.toml section

Reply:
Added a section for the markers